src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -3,6 +3,7 @@
 namespace Illuminate\Database\Eloquent;
 
 use DateTime;
+use Throwable;
```

Comment:
we sort by length, and then alphabetically for the ones that are the same length